### PR TITLE
Manga Clash Url change Fix

### LINKED ISSUE #2537: Mangclash to Toonclash
### Source information

1.4.38

### Source language

English

### Source new URL

https://toonclash.com/

### Other details

When i use on apk, it's not working. Then when i chek for mangaclash.com it become toonclash.com. But the head web name is manga clash. So i think they changed the domain. So i don't know what should i write on the title of this problem. 

### Acknowledgements

- [X] I have updated all installed extensions.
- [X] I have opened WebView and checked that the source URL has not changed yet.
- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

### PR DESCRIPTION
**Pull request by Akshar062**
Fix for issue explained in [Issue #2546](https://github.com/keiyoushi/extensions-source/issues/2546)  and [Issue #2537](https://github.com/keiyoushi/extensions-source/issues/2537) 

closes #2537

Checklist:

- [x] Updated `extVersionCode` value in `build.gradle` for individual extensions
- [ ] Updated `overrideVersionCode` or `baseVersionCode` as needed for all multisrc extensions
- [ ] Referenced all related issues in the PR body (e.g. "Closes #xyz")
- [x] Added the `isNsfw = true` flag in `build.gradle` when appropriate
- [x] Have not changed source names
- [ ] Have explicitly kept the `id` if a source's name or language were changed
- [x] Have tested the modifications by compiling and running the extension through Android Studio
- [ ] Have removed `web_hi_res_512.png` when adding a new extension

Manga Demon fix

Manga Clash url Fix

### CODE REVIEW

File: src/en/mangaclash/build.gradle [hunk 1]
```diff
@@ -3,7 +3,7 @@ ext {
     extClass = '.MangaClash'
     themePkg = 'madara'
     baseUrl = 'https://mangaclash.com'
```
Comment (AwkwardPeak7) [inline src/en/mangaclash/build.gradle]: need to change here too

**Review by AwkwardPeak7 — CHANGES_REQUESTED** [review]
next time please open one pr for one extension

also make sure to write "closes #xyz" for each issue so github links them

also rebase on main

File: src/en/mangaclash/build.gradle (hunk 1 above)
Comment (Akshar062) [reply src/en/mangaclash/build.gradle]: Done Change in [src/en/mangaclash/build.gradle](https://github.com/keiyoushi/extensions-source/pull/2568/files/4c58fe84e99d2d8bbb96913ec0822bbbad76acab#diff-fc16076743a7c9368aa3b82aac278cc1bb46b5d77a91cd48d785c4fc696a2c09)  

Close issue #2546 & #2537 


image

**Comment by Akshar062** [comment]
> next time please open one pr for one extension
> 
> also make sure to write "closes #xyz" for each issue so github links them
> 
> also rebase on main

Ok i have made change and next time i will take this steps into consideration


Done Change in [src/en/mangaclash/build.gradle](https://github.com/keiyoushi/extensions-source/pull/2568/files/4c58fe84e99d2d8bbb96913ec0822bbbad76acab#diff-fc16076743a7c9368aa3b82aac278cc1bb46b5d77a91cd48d785c4fc696a2c09)

Close issue https://github.com/keiyoushi/extensions-source/issues/2546 & https://github.com/keiyoushi/extensions-source/issues/2537


image

**Review by AwkwardPeak7 — APPROVED** [review]

**Comment by AwkwardPeak7** [comment]
you need to put "closes #xyz" in the main body of pr. you can edit it. 
for next time :)

**Pull request merged**